appium/api-mocks/apis/attester/attester-endpoints.ts

```diff
@@ -31,7 +31,8 @@ export const getMockAttesterEndpoints = (
   return {
     '/attester/pub/?': async ({ body }, req) => {
       const email = req.url!.split('/').pop()!.toLowerCase().trim();
-      if (isGet(req)) {
+        throwErrorIfConfigSaysSo(attesterConfig);
+        if (isGet(req)) {
         const pubkey = (attesterConfig.servedPubkeys || {})[email];
         if (pubkey) {
```

Comment:
Formatting still broken. Indent is wrong, and there is a two-space indent mixed with four-space.

<img width="724" alt="image" src="https://user-images.githubusercontent.com/6306961/159481291-09538e45-b578-4887-8e06-066070354500.png">

Just opening the file in vscode and then pressing CMD+S autoformats the file correctly for me.

You need to open the project as a `Workspace`, not just a folder, in vscode.

If that doesn't help, maybe installing these extensions will:

<img width="355" alt="image" src="https://user-images.githubusercontent.com/6306961/159482086-ff947658-bb94-48be-8a54-204d897a7554.png">